### ISSUE TITLE
APT whitelist request for autoconf

### ISSUE THREAD

**Issue by arr2036** [travis-ci/travis-ci#3427]
Needed for generating configure scripts from autoconf.ac files

**Comment by BanzaiMan** [comment]
https://travis-ci.org/BanzaiMan/travis_production_test/builds/55436063#L109

It should already be available for you. Do you have a build log URL that indicates otherwise?

**Comment by arr2036** [comment]
Yes: https://travis-ci.org/arr2036/freeradius-server/jobs/55338462

**Comment by BanzaiMan** [comment]
Where do you call `autoconf` in that job? It shows that you cannot install `autoconf`. `./configure` is failing, because it is unable to find the library `libtalloc`. https://travis-ci.org/arr2036/freeradius-server/jobs/55338462#L215 That is completely different.

**Comment by arr2036** [comment]
I'm aware of that. It's called significantly later as part of the package building/testing process.

I added a ticket to get autoconf white listed, because it's the start of the big long list of packages I need to get white listed before we can move to the container based infrastructure.

If autoconf is actually installed and available, may I make the suggestion that the apt package installer NOT report a failure installing it, and note that it's already provided.

**Comment by BanzaiMan** [comment]
Please open a separate ticket about that. I'm closing this one, since this is about `autoconf`.

**Comment by BanzaiMan** [comment]
Also, I've listed all the packages installed on the current image in an earlier comment, so that you don't have to check for each one. (This list should be valid until the next build image update.)

**Comment by arr2036** [comment]
Yes, I saw that, thanks.
